README.md

```diff
@@ -939,8 +939,13 @@ class Banana < Fruit
 end
 ```
 
-However, there is a known issue when reifying [associations](#associations),
-see https://github.com/paper-trail-gem/paper_trail/issues/594
+A change in what `item_type` stores for subclassed models was introduced in v9.2, recording
```

Comment:
9.2 has been released already, so please reference the PR instead of a version number.